src/utils/serve-functions.js

```diff
@@ -430,28 +435,37 @@ const getFunctionsServer = async function ({ dir, omitFileChangesLog, siteUrl, c
 }
 
 const getBuildFunction = ({ functionBuilder, log }) =>
-  async function build(updatedPath) {
-    log(
-      `${NETLIFYDEVLOG} Function builder ${chalk.yellow(functionBuilder.builderName)} ${chalk.magenta(
-        'building',
-      )} functions from directory ${chalk.yellow(functionBuilder.src)}`,
-    )
+  async function build(updatedPath, eventType) {
+    const relativeFunctionsDir = relative(cwd(), functionBuilder.src)
```

Comment:
This is a bit outside of the scope of this PR, and I'm happy to break it out if it makes it easier to review, but I thought we could show the function path relative to the functions directory as opposed to showing the full path, which is a bit heavy on the output.